app/screens/phone-auth-screen/phone-login-input.tsx

```diff
@@ -124,16 +125,14 @@ export const PhoneInputScreen: React.FC = () => {
     setCountryCode,
     supportedCountries,
     loadingSupportedCountries,
-  } = useRequestPhoneCode({
-    skipRequestPhoneCode: appConfig.galoyInstance.name === "Local",
```

Comment:
Was this intentional to remove the local bypass? How will a user go through the phone flow locally?

Reply:
It's still here, just not on the screen but in the hook